.github/workflows/ci_integration.yml

```diff
@@ -146,6 +146,10 @@ jobs:
         uses: "networktocode/gh-action-setup-poetry-environment@v2"
         with:
           python-version: "${{ matrix.python-version }}"
+      - name: "Remove Installed Nautobots (Poetry Workaround)" # Workaround for https://github.com/python-poetry/poetry/issues/2918
+        run: "for i in $(ls -d1 ~/.cache/pypoetry/virtualenvs/*/lib/*/site-packages/nautobot*.dist-info); do poetry run pip uninstall -y nautobot; done"
```

Comment:
Also `nautobot-*.dist-info` (added `-`) would be slightly safer than `nautobot*.dist-info` as it will only match `nautobot-<version>` and wouldn't match any plugins like `nautobot_foo-<version>` - may not be needed at this time but it wouldn't hurt.

Reply:
Will do.